src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int bothside;
+    private int height;
+
+    public RightTriangle(String color, int height, int bothside) {
```

Comment:
According to the checklist, creating a figure should not pass expressions in the constructor. Create separate variables for 'height' and 'bothside' and pass them on for better code readability.